### DISCUSSION TITLE
How to use custom SSL cert and key

### DISCUSSION THREAD

**Discussion by sephtin** [zwave-js/zwave-js-ui#2892] [Q&A] [answered]
I've enabled https, and it works with the self signed cert.
The documentation specifies that enabling https will gen a cert/key if one is not specified.. but not where to specify my cert/key(?).

If I copy or expose my my cert/key to the docker container, where can I specify what cert/key to use?

**Comment by sephtin** [comment]
After digging around in the docker image, I found that it just generates the files in the store dir (cert.pm and key.pm).  I just softlinked them to my certificate, and it's working correctly now.

I'm set, thanks!

**Comment by robertsLando** [comment, marked as answer]
You can also reference to those files using [env vars](https://zwave-js.github.io/zwave-js-ui/#/guide/env-vars) check `SSL_KEY` and `SSL_CERTIFICATE`
